modules/appeals_api/app/controllers/concerns/appeals_api/openid_auth.rb

```diff
@@ -55,6 +57,11 @@ def validate_auth_token!
 
     private
 
+    def unsafe_mode?
+      Rails.env.development? && \
```

Comment:
Any time I see rails environment checks like this I feel compelled to remind folk that all our server environments run rails production mode - even dev-api.va.gov. You'll need to use `Settings.vba_environment` if you want to run in unsafe mode on lower environments (e.g. `Settings.vsp_environment != 'production'`)

But if a rails environment check was what you intended, carry on! :)

Reply:
I believe it was intended for use in the local environment only (to make local testing easier).